packit_service/worker/fedmsg_handlers.py

```diff
@@ -76,14 +76,14 @@ def get_copr_build_url(event: CoprBuildEvent) -> str:
 
 def copr_url_from_event(event: CoprBuildEvent):
     """
-    Get url to builder-live.log.gz bound to single event
+    Get url to builder-live.log bound to single event
```

Comment:
Maybe some note about the redirect?

Reply:
You nitpicker :)

Reply:
Sorry. I know that I will forget that...